### ISSUE TITLE
[bug]: can't login as "god"

### ISSUE THREAD

**Issue by fabfischer** [makeplane/plane#4077]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current behavior

I host my own instance (v0.16-dev, using setup.sh-script/docker).
I only have one user on the instance, who also acts as "God". After a Firefox update, I was logged out and have been unable to log back in ever since. I keep seeing this screen:
![CleanShot 2024-03-27 at 09 48 03@2x](https://github.com/makeplane/plane/assets/1053032/a130d007-c691-4d03-a643-4d7563a22a42)

When I enter the email address of the "God" user, I get the following message:
```
Error! 
New account creation is disabled. Please contact your site administrator.
```
(account creation is actually deactivated by me)

When I try to access `/god-mode`, I am redirected back to the page above, and the URL then is: `https://my-instance.com/?next_path=/god-mode`

How can I log back in and use the tool?"

### Steps to reproduce

not actually sure, what triggered this behavior, but I assume:
1. hosting an instance
2. create god-user
3. log-out
4. try to log-in again

### Environment

Production

### Browser

None

### Version

Self-hosted

**Comment by FutureBuddha** [comment]
I have a similar Situation just that i get the error "The Payload is not valid" and i can not log in in my Account.

**Comment by vihar** [comment]
@FutureBuddha, @fabfischer, sorry for not getting back earlier. Are you still seeing the issue? We’ve made several improvements to `/god-mode`, including fixing authentication and redirection. Please let us know. Thanks!

**Comment by FutureBuddha** [comment]
i found out that the issue come if you start your email with a capital letter. Then it does not work. i just tested it again and it is the same situation.

**Comment by fabfischer** [comment]
> i found out that the issue come if you start your email with a capital letter. Then it does not work. i just tested it again and it is the same situation.

Yes, I had the same problem. I registered with an email which was something like private**_p_**lane@mydomain.com (capital letter). But I was unable to log in with this email. When I changed it to "private**_p_**lane@mydomain.com" (i.e. with a small "p"), I was able to login.

So I will close the issue because I found the reason for the described behavior. Thanks for your help anyway.
